modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/jms/utils/JMSConstants.java

```diff
@@ -1,84 +0,0 @@
-/*
```

Comment:
Was thinking of removing duplicate constants. As it is better to make Ballerina independent of transport we use, will add them again. Thanks for pointing this out.